### PR TITLE
json_grpc_transcoder: add missing dependency with rbe

### PR DESCRIPTION
**Pull request by TAOXUY**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description: https://github.com/envoyproxy/envoy/issues/9335
Risk Level: Low
Testing: None
Docs Changes: None
Release Notes: None

### CODE REVIEW

File: source/extensions/filters/http/grpc_json_transcoder/BUILD [hunk 1]
```diff
@@ -21,6 +21,7 @@ envoy_cc_library(
         "grpc_transcoding",
         "http_api_protos",
         "api_httpbody_protos",
+        "rpc_status_proto",
```
Comment (qiwzhang) [inline source/extensions/filters/http/grpc_json_transcoder/BUILD]: can we just use "@com_google_googleapis//google/rpc:status_cc_proto" directly?

File: bazel/api_binding.bzl [hunk 2]
```diff
@@ -34,3 +34,7 @@ def envoy_api_binding():
         name = "http_api_protos",
         actual = "@com_google_googleapis//google/api:annotations_cc_proto",
     )
+    native.bind(
+        name = "rpc_status_proto",
+        actual = "@com_google_googleapis//google/rpc:status_cc_proto",
+    )
```
Comment (dio) [inline bazel/api_binding.bzl]: From: `./tools/check_format.py fix bazel/api_binding.bzl` output

```
diff --git a/bazel/api_binding.bzl b/bazel/api_binding.bzl
index 5f7e105c7..ceed11b50 100644
--- a/bazel/api_binding.bzl
+++ b/bazel/api_binding.bzl
@@ -37,4 +37,4 @@ def envoy_api_binding():
     native.bind(
         name = "rpc_status_proto",
         actual = "@com_google_googleapis//google/rpc:status_cc_proto",
-    )
\ No newline at end of file
+    )
```

**Comment by dio** [comment]
Hum, seems like it is related to #9338.

**Comment by zuercher** [comment]
I think if you merge master, the bazel fuzzit pipeline will be removed and this should pass.

**Pull request closed without merging**